less/index.less

```diff
@@ -57,3 +57,105 @@ html, body {
     height: 100%;
 }
 
+@important: red;
+
+@writing: Arial;
```

Comment:
I would watch my naming convention here. Call these `important-text-color` for instance and `font` or `reading-font`